webpack/assets/javascripts/react_app/redux/API/APIActions.js

```diff
@@ -14,4 +15,20 @@ const { GET } = API_OPERATIONS;
  * @param { Object } payload.payload the API payload which will be passed also to the reducer.
  * @param { Object } payload.actionTypes action types which will replace the default action types.
  */
-export const get = payload => ({ type: GET, payload });
+export const apiAction = (type, payload) => ({ type, payload });
+
+export const get = payload => apiAction(GET, payload);
+
+export const post = payload => apiAction(POST, payload);
+
+export const put = payload => apiAction(PUT, payload);
+
+export const patch = payload => apiAction(PATCH, payload);
+
+export const APIActions = {
+  get,
+  post,
+  put,
+  patch,
+  delete: payload => apiAction(DELETE, payload),
```

Comment:
Should also be a const like the rest maybe with a different name like `apiDelete`?

Reply:
it's possible, though it's not an API request type..
I would prefer if someone uses it as:
```js
import { APIActions } ...
APIActions.delete()
```